src/core/public/http/fetch.ts

```diff
@@ -53,6 +53,7 @@ interface Params {
 
 const JSON_CONTENT = /^(application\/(json|x-javascript)|text\/(x-)?javascript|x-json)(;.*)?$/;
 const NDJSON_CONTENT = /^(application\/ndjson)(;.*)?$/;
+const EVENT_STREAM_CONTENT = /text\/event-stream/;
```

Comment:
nit: better to align with json and ndjson defined above? `/^(text\/event-stream)(;.*)?$/;`

Reply:
Sure, updated.